impexp-client/build.gradle

```diff
@@ -141,6 +141,21 @@ distributions.main {
     }
 }
 
+distributions {
+    minimized {
+        distributionBaseName = project.appName + '-Minimized'

```

Comment:
I suggest to leave `project.appName` as it is. This changes the output folder and therefore breaks the `Dockerfiles`.
If there is a good reason the the changed naming it not a big deal to change this in the `Dockerfiles`though.